docs/guides/conda.rst

```diff
@@ -8,16 +8,171 @@ and need them installed when building their documentation.
+  cross-platform compatibility, however it may include packages that are not needed to build your docs.
+- And finally, you can also write it by hand. This allows you to pick exactly the packages needed to
+  build your docs (which also results in faster builds) and overcomes some limitations in the conda
+  exporting capabilities.
+
+For example, using the second method for an existing environment:
+
+.. code-block::
+
+    $ conda activate rtd38
+    (rtd38) $ conda env export --from-history | tee environment.yml
+    name: rtd38
+    channels:
+      - defaults
+      - conda-forge
+    dependencies:
+      - rasterio==1.2
+      - python=3.8
+      - pytorch-cpu=1.7
+    prefix: /home/docs/.conda/envs/rtd38
+
+Read the Docs will override the ``name`` and ``prefix`` of the environment when creating it,
+so they can have any value, or not be present at all.
+
+.. tip:: Bear in mind that ``rasterio==1.2`` (double ``==``) will install version ``1.2.0``,
+   whereas ``python=3.8`` (single ``=``) will fetch the latest ``3.8.*`` version,
+   which is ``3.8.8`` at the time of writing.
+
+.. warning:: Pinning Sphinx and other Read the Docs core dependencies
+   is not yet supported by default when using conda (see `this GitHub issue for discussion`_).
+   If your project needs it, request that we enable the ``CONDA_APPEND_CORE_REQUIREMENTS``
+   :ref:`feature flag <guides/feature-flags:Feature Flags>`.
+
+.. _this GitHub issue for discussion: https://github.com/readthedocs/readthedocs.org/issues/3829
+.. _exporting a conda environment: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment
+
+Effective use of channels
+-------------------------
+
+Conda packages are usually hosted on https://anaconda.org/, a registration-free artifact archive
+maintained by Anaconda Inc. Contrary to what happens with the Python Package Index,
+different users can potentially host the same package in the same repository,
+each of them using their own **channel**. Therefore, when installing a conda package,
+conda also needs to know which channels should use, and which ones take precedence.
+
+If not specified, conda will use ``defaults``, the channel maintained by Anaconda Inc.
+and subject to `Anaconda Terms of Service`_. It contains well-tested versions of the most widely used
+packages. However, some packages are not available on the ``defaults`` channel,
+and even if they are, they might not be on their lastest versions.
+
+As an alternative, there are channels maintained by the community that have a broader selection
+of packages and more up-to-date versions of them, the most popular one being ``conda-forge``.
+
+To use the ``conda-forge`` channel when specifying your project dependencies, include it in the list
+of ``channels`` in ``environment.yml``, and conda will rank them in order of appearance.
+To maximize compatibility, we recommend putting ``conda-forge`` above ``defaults``.
```

Comment:
Could use a code snippet showing this here.

Reply:
Done!